notebooks/community/CODEOWNERS

```diff
@@ -19,5 +19,4 @@
 /neo4j @benofben @htappen
 /sdk/pytorch_lightning_custom_container_training.ipynb @brianchunkang
 /tensorboard @yfang1 @wattli
-/feature_store @nayaknishant @morgandu
-/vertex_endpoints/tf_hub_obj_detection/deploy_tfhub_object_detection_on_vertex_endpoints.ipynb @jfacevedo
+/feature_store @nayaknishant @morgandu @inardini
```

Comment:
Can you add a new line to point directly to this new folder?

Reply:
You don't need to own all the feature_store notebooks, just this specific folder.